src/librustdoc/clean/utils.rs

```diff
@@ -156,39 +156,7 @@ pub(super) fn external_path(
     }
 }
 
-crate fn strip_type(ty: Type) -> Type {
-    match ty {
-        Type::ResolvedPath { path, did } => Type::ResolvedPath { path: strip_path(&path), did },
```

Comment:
Wait, was this never called on anything besides `ResolvedPath` before ??

Reply:
I guess not :)

It was actually only used in a single place before.